kubernetes/uptime-kuma/ingressroute.yml

```diff
@@ -0,0 +1,39 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    nginx.ingress.kubernetes.io/proxy-read-timeout: "3600"
+    nginx.ingress.kubernetes.io/proxy-send-timeout: "3600"
+    nginx.ingress.kubernetes.io/server-snippets: |
```

Comment:
ingress-nginx is already setting all these by default. Not sure if you use ingress-nginx, or nginx-ingress (which are ocnfusingly two distinct projects)

Reply:
to be honest, i would see it in there because i had issues running another websocket based project without these settings